File: python/exercício_33.py
print()
print("=-"*60)
print()
print("--------------Coodernadas------------------")
print()
print("=-"*60)
print() 
x = int(input("Digite um valor para o X: "))
y= int(input("complete as coordenads: {},:".format(x)))

if x == 0:
    if y == 0:
        print("As coordenadas, {},{} se encontram no eixo".format(x,y))
if x >= 1:
    if y >=1:
        print("As coordenads {},{} se encontram no primeiro quardrante".format(x,y))
if x <= -1:
    if y >=1 :
        print("As coordenads {},{} se encontram no segundo quardrante".format(x,y))
if x <= -1:
    if y <=-1:
        print("As coordenads {},{} se encontram no terceiro quardrante".format(x,y))
if x >= 1:
    if y <=-1:
        print("As coordenads {},{} se encontram no quartro quardrante".format(x,y))

File: java/exercício_33.java
import javax.swing.JOptionPane;

public class exercício_33 {
    public static void main(String args[]){
        
        float x,y;
    
        x = Float.parseFloat(JOptionPane.showInputDialog("Digite o primeiro valor: "));
        y = Float.parseFloat(JOptionPane.showInputDialog("Digite o segundo valor: "));
       
            if (x == 0.0 && y == 0.0){ 
               JOptionPane.showMessageDialog(null,"Origem\n");
            }else if (x > 0 && y > 0){
                JOptionPane.showMessageDialog(null,"Quadrante 1\n");
            }else if (x < 0 && y > 0){
               JOptionPane.showMessageDialog(null,"Quadrante 2\n");
            }else if (x < 0 && y < 0){
                JOptionPane.showMessageDialog(null,"Quadrante 3\n");
            }else if (x > 0 && y < 0){
                JOptionPane.showMessageDialog(null,"Quadrante 4\n");
            }else if (x == 0){
                JOptionPane.showMessageDialog(null,"Eixo Y");
            }else if (y == 0){
                JOptionPane.showMessageDialog(null,"Eixo X"); 
            }
        }
}
